src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -29,16 +34,24 @@ public Long elementCount(List<String> elements, String element) {
      * Вернуть Optional первого элемента коллекции</p>
      **/
     public Optional<String> firstElement(List<String> elements) {
-        return null;
+        Optional<String> firstElement = elements.stream().findFirst();
+        return firstElement;
     }
 
     /**
      * <p>4. Дана коллекция строк List of String elements
      * (приметр: Arrays.asList(«a1», «a2», «a3», «a1»)).
-     * Найти элемент в коллекции равный `element` или кинуть ошибку NoSuchElementException</p>
+     * Найти элемент в коллекции равный `element`
+     * или кинуть ошибку NoSuchElementException</p>
      **/
     public String findElement(List<String> elements, String element) {
-        return null;
+        String result = "";
+        if (elements.contains(element)) {
+            result = element;
+        } else {
```

Comment:
@SviatoslavExpert Please try avoid using if else where it is possible and use Stream API instead